esp-link/cgiwifi.c

```diff
@@ -26,6 +27,27 @@ Cgi/template routines for the /wifi url.
 #define DBG(format, ...) do { } while(0)
 #endif
 
+#define LOG(severity, format, ...)  do {	\
+	char buffer[128];			\
+	os_sprintf(buffer, format, ## __VA_ARGS__);	\
```

Comment:
Had a look at the disassemled code. All strings are in ".section .rodata.str1.1, "MSa", @progbits, 1". 
So I assumed they are in flash also...